lib/super_digit.rb

```diff
@@ -1,15 +1,28 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n) where n is the number of digits
+# log10(n) where n is the value of n
+# Space Complexity - O(n) where n is the value of n
```

Comment:
Since you only recur log <sub>10</sub> n times, I think the space complexity is the same as time.